src/BPEssentials/Commands/Home/Home.cs

```diff
@@ -0,0 +1,50 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BrokeProtocol.Entities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace BPEssentials.Commands
+{
+    public class Home : Command
+    {
+        public void Invoke(ShPlayer player, int homeNumber)
+        {
+
+            List<ShApartment> apartments = new List<ShApartment>(player.ownedApartments.Keys);
```

Comment:
Is this really needed?

Reply:
Ik its stupid, but no other way works lol (maybe arrays will work)